### PR TITLE
Initial placement

### PR DESCRIPTION
**Pull request by pahp**
Robot is randomly placed on top of a base, which is persistent. Robot can still eat things like apples. Things get weird when it goes to a new level, so I need to gut that behavior.

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -30,29 +30,49 @@ class AppleFinder(GridGame):
...
         self.in_pit = False
-        centerx = self.MAP_WIDTH // 2
-        centery = self.MAP_HEIGHT // 2
-        self.player_pos = [centerx, centery]
+
+
+        # place robot and base randomly
+        x = self.random.randint(0, self.MAP_WIDTH - 1)
+        y = self.random.randint(0, self.MAP_HEIGHT - 1)
+        self.base_bos = [x, y]
```
Comment (derpferd) [inline game.py]: Did you mean `base_pos`? If not I am not sure what `bos` is maybe add a comment.

**Review by derpferd — CHANGES_REQUESTED** [review]
Just one comment.

File: game.py (hunk 1 above)
Comment (pahp) [reply game.py]: self.base_bos is a typo that is never used... can I edit it in this pull request?

Comment (derpferd) [reply game.py]: I believe so but I am not sure

**Review by derpferd — APPROVED** [review]
LGTM

File: game.py (hunk 1 above)
Comment (pahp) [reply game.py]: I deleted this line. I realize what I was going to use it for (the position of the base), but that is just also the initial placement location, and the base can't move, so I did not need "base_pos" (which I typoed as base_bos). I removed this from the committed file.

**Pull request merged**